cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -119,17 +138,13 @@ def render_files(reldata=None):
                         'etcd_cert': cert})
 
     charm_dir = hookenv.charm_dir()
-    rendered_kube_dir = os.path.join(charm_dir, 'files/kubernetes')
-    if not os.path.exists(rendered_kube_dir):
-        os.makedirs(rendered_kube_dir)
     rendered_manifest_dir = os.path.join(charm_dir, 'files/manifests')
```

Comment:
I see that the manifest_directory was removed on 132. do we still need this block?